package.json

```diff
@@ -1,11 +1,11 @@
 {
-  "name": "test-results-reporter",
-  "version": "1.1.6",
+  "name": "testbeats",
+  "version": "2.0.0",
```

Comment:
If it's a new repo let's reset the version?

Reply:
as we are using the same repo, we already have earlier versions registered in releases. so I bumped the major version.